components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/indexing/indexer/CustomAPIIndexer.java

```diff
@@ -87,7 +87,7 @@ public IndexDocument getIndexedDocument(AsyncIndexer.File2Index fileData) throws
                 registry.put(resourcePath, resource);
             }
 
-            if (storeVisibility != null) {
+            /*if (storeVisibility != null) {
```

Comment:
Why we are keeping the commented out code on the source

Reply:
This has been done as a temporary fix for https://github.com/wso2/product-apim/issues/2666.
We have properly fixed this in https://github.com/wso2/carbon-apimgt/pull/5034